addon-test-support/create.js

```diff
@@ -156,18 +169,19 @@ export function create(definitionOrUrl, definitionOrOptions, optionsOrNothing) {
     options = definitionOrOptions || {};
   }
 
-  definition = assign({}, definition);
+  let { context } = definition;
```

Comment:
`context` seems unused

Reply:
`page.setContext(context);` at the end of `create`.

Reply:
Sure, sorry, missed that 🙈